oci/container.go

```diff
@@ -36,6 +36,7 @@ type Container struct {
 	stdinOnce       bool
 	privileged      bool
 	trusted         bool
+	allowStrace     bool
```

Comment:
Do we want to poke these holes one at a time?  Or do we want to poke a big hole and add:

```go
server  *server.Config
```

so the container can use `c.server.AllowStrace`?  A bigger hole would make the `NewContainer` interface more stable if we need additional server configs passed through later on.